gradle.properties

```diff
@@ -1,6 +1,7 @@
 version=7.0.1
 iexecCommonVersion=5.9.1-NEXT-SNAPSHOT
 iexecBlockchainAdapterVersion=7.0.0-NEXT-SNAPSHOT
+iexecResultVersion=7.0.2-NEXT-SNAPSHOT
```

Comment:
We might switch to a dedicated library version, which should be pretty stable over time. My point is: A new internal result-proxy patch should not make the previous API library version obsolete

Reply:
Wouldn't the service (result-proxy) and its library progress together? Are you suggesting we dissociate them?

Reply:
I have several remarks on this:
* We have never used and published library this way, it's hard to understand the impacts
* If we switch to a dedicated library version, it breaks the idea to manage library and spring boot app in same repository (tags management will become too hard)

In the current topology, if a patch is done on a spring boot app, a new tag/version will be created and will create a new spring boot app as well as a new library. If the library has not been modified, the new library version will have the same functionality level as the previous one. We can stick to the previous library version or propagate it elsewhere

If the library has been modified, we will chose to deploy or not the new spring boot application and will need to use the library in other applications and potentially create new versions for those one.

If we want to split library management for spring boot application management, we must create dedicated repositories for  each libraries and adapt our workflow.

Reply:
I'm not saying to do it now, but we need to think about coupling we are introducing
Something like
iexecResultVersion=x.y.z
iexecResultApiVersion=${iexecResultVersion.major}.${iexecResultVersion.minor}